src/components/ui/Button/Button.styled.js

```diff
@@ -0,0 +1,59 @@
+import styled, { css } from 'styled-components';
+
+const Button = styled.button`
+	border-radius: 2.5px;
+	background: none;
+
+	${({ variant }) =>
+		variant == 'dropUp' &&
+		css`
+			border: none;
+		`};
+
+	${({ variant }) =>
+		variant == 'dropUpSecond' &&
+		css`
+			border: none;
+			background-image: radial-gradient(black 2%, transparent);
+			margin-bottom: 0.5rem;
+		`};
+
+	${({ variant }) =>
+		variant == 'dark' &&
+		css`
+			margin: 1rem;
+			width: 7rem;
+			height: 3rem;
+			background-color: transparent;
+			border: 1px solid var(--color-primary-light);
+			border-radius: 2.5px;
+			box-sizing: border-box;
+			color: var(--color-primary-light);
+			position: relative;
+
+			&:hover,
+			&:active {
+				outline: 0;
+			}
+
+			&:hover {
+				background-color: transparent;
+				cursor: pointer;
+			}
+
+			&:before {
+				background-color: #0d2748;
+				content: '';
+				height: calc(100% + 3px);
+				position: absolute;
+				right: -0.5rem;
+				top: -0.6rem;
+				transition: background-color 300ms ease-in;
```

Comment:
I'm not sure this works as intended. :9 There is no pre-defined background-color on this element, so the transition doesn't trigger.

Reply:
Honestly: I don't know either as I stumbled above this online, thought, it would match my forms and deleted already everything, of which I knew, it would do nothing in my styling... 😄

Reply:
I think you'd see the transition if you included a hover effect for the button, but `:before` is a little whacky sometimes.